manticore/ethereum/solidity.py

```diff
@@ -0,0 +1,191 @@
+import pyevmasm as EVMAsm
+
+from .abi import ABI
+from ..exceptions import EthereumError
+
+
+class SolidityMetadata(object):
```

Comment:
Hmm maybe for future. Rename this to just Metadata? ethereum.solidity.Metadata ?

Reply:
good idea! although i don't particularly mind SolidityMetadata either. having just `Metadata` floating around the codebase elsewhere could be a little vague as to what it refers to (then you need to track it back to where the import came from to see the solidity party)